core/management/commands/load_titles.py

```diff
@@ -15,7 +15,10 @@
 _logger = logging.getLogger(__name__)
 
 class Command(BaseCommand):
-    help = "Load a MARC XML file of title records"
+    help = """
```

Comment:
Oooooh this is a very helpful command that I didn't really know about

Reply:
Yeah, this one saved us a lot of pain this past year.  I no longer have to keep that marc-mirror code updated with a separate branch for UO's overridden MARC files!